app/src/main/java/fr/free/nrw/commons/explore/ExploreListRootFragment.java

```diff
@@ -113,13 +109,14 @@ public void onAttach(final Context context) {
 
   @Override
   public void onMediaClicked(int position) {
-    Log.d("deneme8","on media clicked");
+
```

Comment:
Did you purposefully add these new lines (at the beginning and end of the method)?
I don't think it is the usual convention, right?
By the way, you can easily check all of your changes by using Git's branch comparison tool.

Reply:
I'll remove them,
I was trying code formatting tools and that's the reason for this unintended formatting.
I'll look into it, and take care to recheck before push.
Thanks